src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(int firstLeg, int secondLeg, String color) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        setColor(color);
+        setFigureType("right triangle");
+        calculateArea();
+    }
+
+    @Override
+    public void calculateArea() {
+        setArea(1 / 2 * firstLeg * secondLeg);
+    }
+
+    @Override
+    public String toString() {
+        return super.toString()
+                + getFigureType()
+                + ", area: "
+                + getArea()
+                + " sq.units"
+                + ", first leg: "
+                + firstLeg
+                + " units, second leg: "
+                + secondLeg
+                + " units, color: "
+                + getColor();
```

Comment:
From the requirements:

```
To 'draw' means to print out all information about a figure using System.out.println() (you shouldn't override toString() method for this)
```

So, create an interface with one abstract method `draw()` and implement it in your `Figure`.